src/components/Facility/Consultations/VentilatorTable.tsx

```diff
@@ -0,0 +1,124 @@
+import { formatDateTime } from "@/Utils/utils";
+import { DailyRoundsModel } from "../../Patient/models";
+import { useTranslation } from "react-i18next";
+
+type VentilatorTableProps = {
+  dailyRoundsList?: DailyRoundsModel[];
+};
+
+export default function VentilatorTable(props: VentilatorTableProps) {
+  const { t } = useTranslation();
+  const { dailyRoundsList } = props;
+  let sortedData: DailyRoundsModel[] = [];
+
+  const sortData = (data: DailyRoundsModel[]) => {
+    return data.sort(function (a, b) {
+      const ad = new Date(a.taken_at ?? Date.now());
+      const bd = new Date(b.taken_at ?? Date.now());
+      return ad > bd ? 1 : -1;
+    });
+  };
```

Comment:
checkout the utility fn: `compareBy` or `compareByDateString` right. we could re-use that here.

https://github.com/ohcnetwork/care_fe/blob/develop/src/Utils/utils.ts#L416-L428